### PR TITLE
Add possibility to extract latest version from channel in gke

### PR DESCRIPTION
**Pull request by Argh4k**

### CODE REVIEW

File: kubetest/util.go [hunk 1]
```diff
@@ -216,7 +217,14 @@ func getChannelGKEVersion(project, zone, region, gkeChannel string) (string, err
 
 	for _, channel := range c.Channels {
 		if strings.EqualFold(channel.Channel, gkeChannel) {
-			return "v" + channel.DefaultVersion, nil
+			if len(channel.ValidVersions) == 0 {
+				return "", fmt.Errorf("channel doest not have valid versions")
+			}
+			if strings.EqualFold(extractionMethod, "latest") {
+				return "v" + channel.ValidVersions[0], nil
```
Comment (mborsz) [inline kubetest/util.go:224]: It seems to assume that latest is the first one so the versions.
Do we have any guarantee about the order of ValidVersions in ReleaseChannelConfig?

https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/ServerConfig#releasechannelconfig

**Review by mborsz — COMMENTED** [review]
Looks good, one comment.

File: kubetest/util.go (hunk 1 above)
Comment (Argh4k) [reply kubetest/util.go:224]: Actually, we do not have guarantees about it. I will add some simple sort logic for that.

**Comment by marseel** [comment]
/assign @wojtek-t 
for approval.

**Pull request merged**